Integrations/integration-secdo.yml

```diff
@@ -0,0 +1,135 @@
+description: Secdo's automated incident response platform hunts threats in real time
+  and delivers an endpoint detection and response solution.
+configuration:
+- display: Secdo Server
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Secdo Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Do not validate server certificate (insecute)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Company name
+  name: company
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |+
+    var sendRequest = function() {
+        var requestUrl = 'https://' + params.server + '/publicapi/v2/load_ioc/';
+        var data = {
+            company: params.company,
+            apikey: params.key,
+            severity: args.severity,
+            ioc_data: args.data,
+            source: 'Demisto',
+            response_plan: args.response,
+            artifacts_type: args.artifact,
+            comment: args.comment
+        };
+
+        var res = http(
+            requestUrl,
+            {
+                Method: 'POST',
+                Headers: {
+                    Accept: ['application/json'],
+                    'Content-Type': ['application/json']
+                },
+                Body: JSON.stringify(data)
+            },
+            params.insecure,
+            params.proxy
+        );
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        return res;
+    };
+
+
+    var addIocs = function() {
+        var md = "## Secdo\n";
+        var res =  sendRequest();
+        if (res.StatusCode === 200) {
+            md += 'The IOCs have been successfully added to Secdo.\n';
+        } else {
+            md += 'Opps, it seems that there is some error with adding the IOCs to Secdo.\n'
+        }
+
+        return ( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': {}, "HumanReadable": md} );
+    };
+
+    switch (command) {
+        case 'test-module':
+            return 'ok';
```

Comment:
you should add actual check...

Reply:
The API doesn't have checks. Only submitting new IOCs.

Reply:
Mmmm maybe there is another option to test connectivity..
Maybe add wrong ioc and check that the error you get is the one you expecting for (and not connection error)

Reply:
There is no wrong IOC. It just pushes every text that you write to it.

Reply:
Let's talk about this one

Reply:
Let's talk. Where are you?